hack/run-go-test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
this file needs the boilerplate header (which is why it's failing the unit/integration jenkins suite right now).